Java/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Remove personal VS Code launch file

Reply:
Sure happy to remove, I left it in as hopefully have a nice starting point for people wanting to run the sample. Mean you can pull the repo, open in VSCode and F5 to start things up